gdsfactory/cell.py

```diff
@@ -167,43 +147,36 @@ def _cell(*args, **kwargs):
             f"{key}={clean_value_name(changed2[key])}" for key in sorted(changed.keys())
         ]
 
-        # get only the args which are explicitly passed and different from defaults
-        # if any args were different from default, append a hash of those args.
-        # else, keep only the base name
-        # print(named_args_string)
-
-        if active_pdk.cell_decorator_settings.naming_style == "updk":
+        if naming_style == "updk":
             full_args_list = [
                 f"{key}={clean_value_name(full[key])}" for key in sorted(full.keys())
             ]
             named_args_string = ",".join(full_args_list)
             name = f"{prefix}:{named_args_string}" if named_args_string else prefix
             name = clean_name(name, allowed_characters=[":", ".", "="])
 
-        elif active_pdk.cell_decorator_settings.naming_style == "default":
+        elif naming_style == "default":
             changed_arg_set = set(passed_args_list).difference(default_args_list)
             changed_arg_list = sorted(changed_arg_set)
             named_args_string = "_".join(changed_arg_list)
-            if changed_arg_list or include_module:
-                if include_module and changed_arg_list:
-                    named_args_module_string = f"{named_args_string}_{func.__module__}"
-                elif include_module:
-                    named_args_module_string = func.__module__
-                elif changed_arg_list:
-                    named_args_module_string = named_args_string
-
-                named_args_module_string = (
-                    hashlib.md5(named_args_module_string.encode()).hexdigest()[:8]
+
+            if include_module:
+                named_args_string += f"_{func.__module__}"
```

Comment:
[No change needed] If I wanted the module name to go _before_ the function name, such that the cached name was `{module}.{func}_{args_suffix}`, how would I go about that?

Reply:
You would need to create your own cell decorator